dosed/datasets/dataset.py

```diff
@@ -105,11 +108,33 @@ def __init__(self,
         data = Parallel(n_jobs=n_jobs, prefer="threads")(delayed(get_data)(
             filename="{}/{}".format(h5_directory, record),
             signals=signals,
-            fs=fs
-        ) for record in self.records)
+            fs=fs,
+            window=self.window
+        ) for record in tqdm.tqdm(self.records))
```

Comment:
tqdm in parallel ? Does it output correct outputs in parallele mode ?

Reply:
No it doesn't.
It only shows the time when a record starts being processed and not the time when it is finished.
I will find a workaround.